### PR TITLE
[BUGFIX] `ColumnPairExpectation` tests need to consider 2 possible GroupBy results

### PR DESCRIPTION
**Pull request by Shinnnyshinshin**
- Pandas `groupby()` can output 2 possibilities for `ColumnPairExpectations`. Tests now use a `or` condition to test either possibility

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: tests/render/test_util.py
Comment (NathanFarmer) [inline tests/render/test_util.py:391]: `res.index.to_list() in ["('eraser', 'desk')", "('desk', 'eraser')"]`?

**Review by NathanFarmer — APPROVED** [review]
Non-blocking suggestion

File: tests/render/test_util.py
Comment (Shinnnyshinshin) [reply tests/render/test_util.py:391]: oo I like it. I'll apply the changes before merging 🙇🏼

Comment (Shinnnyshinshin) [reply tests/render/test_util.py:391]: update: the `in` comparison doesn't seem to be as straightforward.  Merging, so I'm not holding up `develop`.

**Pull request merged**
